pkg/plugins/archfilter/archfilter.go

```diff
@@ -114,6 +114,13 @@ func GetPodArchitectures(pod *v1.Pod) ([]ImageArch, error) {
 	containers := []ImageArch{}
 	for _, container := range pod.Spec.Containers {
 		val, err := FetchFromCache(cacheStore, container.Image)
+
+		// Skip Istio proxy containers
```

Comment:
Please make this configurable, e.g. SKIP_CONTAINER_NAMES, so this can be contributed upstream cleanly